cdc/tests/integration_tests/flow_control/run.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+set -euo pipefail
+
+CUR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+source $CUR/../_utils/test_prepare
+WORK_DIR=$OUT_DIR/$TEST_NAME
+CDC_BINARY=tikv-cdc.test
+SINK_TYPE=$1
+UP_PD=http://$UP_PD_HOST_1:$UP_PD_PORT_1
+DOWN_PD=http://$DOWN_PD_HOST:$DOWN_PD_PORT
+
+function run() {
+	rm -rf $WORK_DIR && mkdir -p $WORK_DIR
+
+	start_tidb_cluster --workdir $WORK_DIR
+
+	cd $WORK_DIR
+
+	start_ts=$(tikv-cdc cli tso query --pd=$UP_PD)
+	sleep 10
+
+	cat - >"$WORK_DIR/tikv-cdc-config.toml" <<EOF
+per-changefeed-memory-quota=102400 #100K
+[sorter]
+max-memory-consumption=0
+EOF
+
+	export GO_FAILPOINTS='github.com/tikv/migration/cdc/cdc/processor/pipeline/ProcessorSinkFlushNothing=1000*return(true)'
+	run_cdc_server --workdir $WORK_DIR --binary $CDC_BINARY --config $WORK_DIR/tikv-cdc-config.toml
+	rss0=$(ps -aux | grep 'tikv-cdc' | head -n1 | awk '{print $6}')
+	if [[ $rss0 == "" ]]; then
+		exit 1
+	fi
+
+	case $SINK_TYPE in
+	tikv) SINK_URI="tikv://${DOWN_PD_HOST}:${DOWN_PD_PORT}" ;;
+	*) SINK_URI="" ;;
+	esac
+
+	tikv-cdc cli changefeed create --start-ts=$start_ts --sink-uri="$SINK_URI"
+	rawkv_op $UP_PD put 100000 # About 30M
+
+	rss1=$(ps -aux | grep 'tikv-cdc' | head -n1 | awk '{print $6}')
+	if [[ $rss1 == "" ]]; then
+		exit 1
+	fi
+	expected=1048576 # 1M
```

Comment:
The unit of `RSS` is kb (see [`ps`](https://man7.org/linux/man-pages/man1/ps.1.html)).
So it's actually `1GB` here.

>       rss         RSS       resident set size, the non-swapped physical
>                             memory that a task has used (in kilobytes).
>                             (alias rssize, rsz).

And I saw that this [CI log](https://ci.pingcap.net/blue/rest/organizations/jenkins/pipelines/common-check/runs/101816/nodes/47/steps/48/log/?start=0) said "cdc server used memory: 381204".

I think even if flow controller is working as expected, the tikv-cdc server will consume such hundreds of MB memory (for other running objects). So maybe we need to put more data (maybe several GB).

Reply:
Thank you for your reminder, your reminder is is important for this test case. I redesign this test case, PTAL~